packages/components/Callout/macos/CalloutWindow.swift

```diff
@@ -47,7 +68,12 @@ class CalloutWindow: NSWindow {
 
 	@objc private func dismissCallout() {
 		lifeCycleDelegate?.calloutWillDismiss(window: self)
-		close()
+
+		if let monitor = mouseEventMonitor {
+			NSEvent.removeMonitor(monitor)
+		}
+
+		orderOut(self)
```

Comment:
Calling orderOut: "causes the window to be removed from the screen, but does not cause it to be released." so when does it actually release?

Reply:
Presumably when the placeholder view `Callout` (What is defined in `Callout.Swift`) that owns the window is deallocated, since Callout is one of it's private variables. React Native manages the ownership of the placeholder view. I wanted the window to be allocated as long as it's placeholder view was allocated.